prime-router/src/main/kotlin/serializers/Hl7Serializer.kt

```diff
@@ -256,7 +270,11 @@ class Hl7Serializer(val metadata: Metadata) {
         val mappedRows = mapping.mappedRows
         errors.addAll(mapping.errors.map { ResultDetail(ResultDetail.DetailScope.ITEM, "", it) })
         warnings.addAll(mapping.warnings.map { ResultDetail(ResultDetail.DetailScope.ITEM, "", it) })
-        return ReadResult(Report(schema, mappedRows, source), errors, warnings)
+        mappedRows.forEach {
+            println("${it.key} -> ${it.value.joinToString()}")
+        }
+        val report = Report(schema, mappedRows, source)
+        return ReadResult(report, errors, warnings)
     }
```

Comment:
Spit out the mapped values on read. We should turn this into a debugging statement.

Reply:
Is this the evil line of code that makes my builds like 10,000 lines long?? ;)